src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,13 @@
+environment=DEV
+###########################################################
+# Web application properties
+###########################################################
+host=https://www.onliner-dev.by/
+###########################################################
+# Files path properties
+###########################################################
+logPath=target/logs/appTest.log
```

Comment:
Are these two parameters different for all the envs or common?

Reply:
Back this two parameters in to Constant